.prettierrc

```
@@ -0,0 +1,4 @@
+{
+    "singleQuote": true,
+    "printWidth": 120
```

```suggestion
  "singleQuote": true,
  "printWidth": 120
```

Please change the indentation to use 2 spaces.